serverlessworkflow/modules/ROOT/pages/cloud/quarkus/deploying-on-openshift.adoc

```diff
@@ -0,0 +1,218 @@
+= Deploying your {product_name} application on OpenShift
+:compat-mode!:
+// Metadata:
+:description: Deploying Serverless Application on Minikube
+:keywords: kogito, workflow, quarkus, serverless, kn, oc, openshift
+:table-caption: Data Set
+// envs for common content
+:registry: OpenShift's
+:cluster_kind: OpenShift with Red Hat OpenShift Serverless is ready
+:k8s_registry: image-registry.openshift-image-registry.svc:5000
+:default_namespace: kogito-serverless
+:command_line_tool: oc
+:command_line_tool_name: OpenShift CLI
+:quarkus-k8s-plugin: quarkus-openshift
+:platform: OpenShift
+:container_push: This property should be removed if deploying on OpenShist Clusters
+// links
+:ocp_local_url: https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.17
+:ocp_local_url_install: https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.17/html/getting_started_guide/installation_gsg
+:ocp_cli_url: https://docs.openshift.com/container-platform/4.12/cli_reference/openshift_cli/getting-started-cli.html#cli-about-cli_cli-developer-commands
+:ocp_kn_cli_url: https://docs.openshift.com/container-platform/4.12/serverless/install/installing-kn.html
+:ocp_knative_eventing_url: https://docs.openshift.com/container-platform/4.12/serverless/install/installing-knative-eventing.html
+:ocp_expose_registry_url: https://docs.openshift.com/container-platform/4.12/registry/securing-exposing-registry.html
+:knative_istio_issue_url: https://access.redhat.com/solutions/4791871
+
+
+
+// ###### Note to the reviewers, should we mention OSL? Or just OpenShift Serverless with Knative is enough?
+
+This document describes how to deploy a {product_name} application using a OpenShift cluster, along with a procedure
+to run the OpenShift Serverless, which is based on the Knative.
+
+.Prerequisites
+* Your {product_name} application is ready to use. +
+For more information about building the application container, +
+see xref:cloud/quarkus/build-workflow-image-with-quarkus-cli.adoc[Building workflow images using Quarkus CLI].
+* OpenShift CLI is installed.+
+For more information, see link:{ocp_cli_url}[Getting started with the OpenShift CLI].
+* Knative CLI is installed. +
+For more information, see link:{ocp_kn_cli_url}[Install the Knative CLI].
+* Knative workflow CLI is installed. +
+For more information see xref:tooling/kn-plugin-workflow-overview.adoc[Serverless Workflow plug-in for Knative CLI].
+* (Optional) Quarkus CLI is installed. +
+For more information, see link:{quarkus_cli_url}[Building Quarkus Apps with Quarkus command line interface (CLI)].
+
```

Comment:
Instead of repeating "For more information", just link the article to the prereq.